### PR TITLE
Support spread/passthrough options on renderers

### PR DESCRIPTION
**Pull request by sandersky**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #none# - documentation fixes and/or test additions
 - [ ] #patch# - backwards-compatible bug fix
 - [x] #minor# - adding functionality in a backwards-compatible manner
 - [ ] #major# - incompatible API change

# CHANGELOG

* **Added** support for passthrough options (spread options). This allows new properties of downstream to be leveraged as soon as they are available, with the downside of them not being validated at the bunsen level.

### CODE REVIEW

File: tests/validator/view-schema/v2-test.js [hunk 1]
```diff
@@ -32,79 +31,177 @@ describe('v2 schema validation', () => {
...
+        it('passes validation when value is number', function () {
+          value.cells[0].renderer.none.value = 1.1
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+
+        it('passes validation when value is integer', function () {
+          value.cells[0].renderer.none.value = 1
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+        it('passes validation when value is boolean', function () {
```
Comment (job13er) [inline tests/validator/view-schema/v2-test.js]: looks like this `it` is missing some blank lines around it

File: tests/validator/view-schema/v2-test.js [hunk 2]
```diff
@@ -32,79 +31,177 @@ describe('v2 schema validation', () => {
...
+        it('passes validation when value is number', function () {
+          value.cells[0].renderer.options.none.value = 1.1
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+
+        it('passes validation when value is integer', function () {
+          value.cells[0].renderer.options.none.value = 1
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+        it('passes validation when value is boolean', function () {
```
Comment (job13er) [inline tests/validator/view-schema/v2-test.js]: Same here with padding around the `it` block

**Review by job13er — CHANGES_REQUESTED** [review]
Pretty nit-picky, just some missing whitespace. Other than that, looks good :+1:

File: tests/validator/view-schema/v2-test.js (hunk 2 above)
Comment (sandersky) [reply tests/validator/view-schema/v2-test.js]: Agreed, I copy/pasted code written by @sophypal and that isn't something we lint so I've not been to pressing on it. I'm also not sure if we've ever documented that as part of our styling standards or not.

**Review by job13er — APPROVED** [review]

**Comment by job13er** [comment]
Approved


[Approved with PullApprove](https://pullapprove.com/ciena-blueplanet/bunsen-core/pull-request/76/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-blueplanet/bunsen-core)

**Pull request merged**